daffodil-core/src/main/scala/org/apache/daffodil/dpath/Conversions.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.daffodil.dpath
```

Comment:
This file moved to daffodil-core from daffodil-runtime1. Not needed in the runtime.